django/api/services/calculate_rebate.py

```diff
@@ -0,0 +1,70 @@
+from ..models.go_electric_rebate_application import GoElectricRebateApplication
+from ..models.household_member import HouseholdMember
+from ..settings import INCOME_REBATES
+
+
+def calculate_rebate_amount(cra_response, application_id):
+    def check_individual(primary_income):
+        if primary_income is None:
+            return "not approved"
```

Comment:
Sonar cloud is complaining about this - there shouldn't be any text comparisons here for these constants (not approved, A, B, C etc)

Reply:
fixed in new commit! 👍